5calls/app/src/main/java/org/a5calls/android/a5calls/view/OutcomeView.java

```diff
@@ -0,0 +1,105 @@
+package org.a5calls.android.a5calls.view;
+
+import android.content.Context;
+import android.content.res.Configuration;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.AppCompatButton;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.LinearLayout;
+
+import org.a5calls.android.a5calls.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class OutcomeView extends LinearLayout {
```

Comment:
It is my understanding that there could be an indeterminate number of outcomes. I'm not sure how to accomplish that with XML layouts other than having an XML file with a bunch of buttons that I programmatically show/hide. I thought this solution would be a little more elegant.